doc/content/source/materials/InelasticRebar.md

```diff
@@ -0,0 +1,65 @@
+# Inelastic Rebar
+
+!syntax description /Materials/InelasticRebar
+
+## Description
+
+The `InelasticRebar` model implements the J2 plasticity for reinforcement bars in concrete that are modeled using 1D truss elements. The model utilizes a discrete finite element implementation strategy to simplify the computational effort. The axial elongation of the element is obtained from
```

Comment:
implements the J2 -> implements J2

I'd delete this sentence: The model utilizes a discrete finite element implementation strategy to simplify the computational effort. (it's implied in all of our models)